### PR TITLE
DEVDOCS-4780: [update] Installing Stencil, update Windows instructions

### PR DESCRIPTION
**Pull request by bc-traciporter**
Updated Windows setup instructions for installing Stencil CLI

### CODE REVIEW

**Review by jairo-bc — APPROVED** [review]

File: docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx [hunk 1]
```diff
@@ -90,10 +90,10 @@ choco install git
 choco install nvm; nvm install 18.15.0; nvm use 18.15.0
 
 # Install Windows C++ Build Tools (also installs python2)
-npm install -g windows-build-tools --vs2015
+npm install -g windows-build-tools --vs2019
 
 # Tell npm to use python2
```
Comment (slsriehl) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx:95]:
```suggestion
# Tell npm to use python3

```

**Review by slsriehl — CHANGES_REQUESTED** [review]
see comments and suggestions -- should be straightforward to resolve.

**Review by bc-evan-johnson — APPROVED** [review]

File: docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx [hunk 2]
```diff
@@ -73,28 +73,18 @@ There are two methods for installing Stencil CLI and its dependencies on Windows
...
 
 # Install git if you don't have it
 choco install git
 
-#####################################################################################
-# Close PowerShell and re-open as admin
-#####################################################################################
-
-# Install nvm-windows and stencil compatible node.js
+# Install nvm windows and stencil compatible node.js
```
Comment (slsriehl) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx:85]:
```suggestion
# Install nvm windows and stencil-compatible node.js

```

File: docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx [hunk 3]
```diff
@@ -73,28 +73,18 @@ There are two methods for installing Stencil CLI and its dependencies on Windows
 
 
 ### Method 1: Install dependencies using Chocolatey
-If you're not comfortable manually installing and configuring Python and Node.js on Windows, or if you prefer an easy installation option, use the [Chocolatey package manager](https://chocolatey.org/) to install Stencil CLI's dependencies. To do so, [open PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/getting-started/starting-windows-powershell?view=powershell-6) as an administrator, and run the following commands:
+If you're not comfortable manually installing and configuring Python and Node.js on Windows, or if you prefer an easy installation option, use the [Chocolatey package manager](https://chocolatey.org/install) to install Stencil CLI's dependencies. To do so, [open PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/getting-started/starting-windows-powershell?view=powershell-6) as an administrator, and run the following commands:
```
Comment (slsriehl) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx:76]:
```suggestion
If you're not comfortable manually installing and configuring Python and Node.js on Windows, or if you prefer an easy installation option, use the [Chocolatey package manager](https://chocolatey.org/install) to install Stencil CLI's dependencies.  Python and C++ build tools are required to compile some dependencies, but you may already have them installed. The troubleshooting section contains information about which versions Stencil currently uses. ... (run into errors go there)

To install Stencil CLI's dependencies, [open PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/getting-started/starting-windows-powershell?view=powershell-6) as an administrator, and run the following commands:

```

Comment (bc-traciporter) [reply docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx:76]: I will trust the engineer's testing of the installation instructions and not add this clarification.

File: docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx [hunk 4]
```diff
@@ -73,28 +73,19 @@ There are two methods for installing Stencil CLI and its dependencies on Windows
...
```
Comment (slsriehl) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx:76]:
```suggestion
If you prefer an easy installation option, use the [Chocolatey package manager](https://chocolatey.org/install) to install Stencil CLI's dependencies. To do so, [open PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/getting-started/starting-windows-powershell?view=powershell-6) as an administrator, and run the following commands:

```

File: docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx [hunk 5]
```diff
@@ -188,21 +177,6 @@ If you receive an error installing Chocolatey, run the following command to enab
 Set-ExecutionPolicy -ExecutionPolicy RemoteSigned
 ```
```
Comment (slsriehl) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx:179]:
```suggestion

### Visual Studio not found error
If you receive the error "Could not find VS", update to the most current version of Stencil CLI and run it using the version of Node.js indicated in this article.

### Python npm configuration error
If you receive "Error: Could not find any Python installation to use", update to the most current version of Stencil CLI and run it using the version of Node.js indicated in this article.

```

**Review by slsriehl — APPROVED** [review]

**Pull request merged**